api/test/wfl/tools/fixtures.clj

```diff
@@ -193,18 +193,6 @@
    datarepo/delete-snapshots-then-dataset
```

Comment:
Here's where we clean up any snapshots as part of cleaning up a temporary dataset.